### PR TITLE
[OC-1400] Add "service" field in the free message feature

### PR DESCRIPTION
**Pull request by vlo-rte**
Release notes : 

In Tasks section : 

[OC-1400](https://opfab.atlassian.net/browse/OC-1400) : Add "service" field in the free message feature

### CODE REVIEW

File: ui/main/src/app/modules/usercard/usercard.component.ts [hunk 1]
```diff
@@ -184,6 +199,37 @@ export class UserCardComponent implements OnDestroy, OnInit {
...
+                    this.servicePerProcesses.set(processOption.value, group.id);
+                }
+            });
+
+            if (processOptions.length > 0)
+                this.processesPerServices.set(group.id, processOptions);
+        });
+
+        if (this.processOptions.length > numberOfProcessesAttachedToAService)
+            this.serviceOptions.push({value: "", label: ""});
```
Comment (freddidierRTE) [inline ui/main/src/app/modules/usercard/usercard.component.ts:228]: Add a default label : 
English : Base Service 
French : Service de base

File: src/test/utils/karate/businessconfig/postProcessGroups.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    And assert response.groups.length == 2
+    Then match response.groups[0].id == 'service1'
+    Then match response.groups[0].processes[0] == 'defaultProcess'
+    Then match response.groups[0].processes[1] == 'userCardExamples'
+    Then match response.groups[1].id == 'service2'
+    Then match response.groups[1].processes[0] == 'userCardExamples2'
+    Then match response.locale.en.service1 == 'Service number 1'
+    Then match response.locale.en.service2 == 'Service number 2'
+    Then match response.locale.fr.service1 == 'Service numéro 1'
+    Then match response.locale.fr.service2 == 'Service numéro 2'
```
Comment (freddidierRTE) [inline src/test/utils/karate/businessconfig/postProcessGroups.feature:31]: Just check status code is 200 . This way we can easly load new configuration with this feature just on modifying the processgoups.json

**Review by freddidierRTE — CHANGES_REQUESTED** [review]

File: src/test/utils/karate/businessconfig/postProcessGroups.feature (hunk 2 above)
Comment (vlo-rte) [reply src/test/utils/karate/businessconfig/postProcessGroups.feature:31]: Done

File: ui/main/src/app/modules/usercard/usercard.component.ts (hunk 1 above)
Comment (vlo-rte) [reply ui/main/src/app/modules/usercard/usercard.component.ts]: Done

**Pull request merged**